.gitignore

```diff
@@ -34,3 +34,7 @@ node_modules
 .sass-cache/
 public/
 .cache/
+
+# .DS_Store Files
```

Comment:
Ok, so this is an error I made. Yes I agree with addin `.DS_Store` to the `.gitignore` file, but I had an area set aside for custom ignores, I just didn't mark it out correctly.

Lines 1-4 are our custom ignores for this repo, the rest were added by `Gatsby`

Can you add a comment at the top of the file donating custom ignores and move `.DS_Store` there please

Many thanks